committeeoversightapp/models.py

```diff
@@ -368,14 +374,16 @@ def total_percent_avg(self):
     def get_percent_max(self, hearing_type):
         try:
             return round(getattr(self, hearing_type) \
```

Comment:
This is getting a bit tough to parse. Can you pull out the `getattr` calls into variables?

```python
ht = getattr(self, hearing_type)
ht_max = getattr(self.committee, hearing_type + '_max')

return round(ht / ht_max * 100 * self.length_handicap)
```

Reply:
yep!